src/components/NcAppContent/NcAppContent.vue

```diff
@@ -268,6 +279,24 @@ export default {
 				},
 			}
 		},
+
+		realPageTitle() {
+			if (this.pageTitle) {
+				return this.pageTitle
+			}
+			if (this.pageHeading) {
+				return t('{view} - {app} - {product}', { view: this.pageHeading, app: appName, product: productName })
+			}
```

Comment:
`appName` seems to be a non-translated constant from the `package.json` file. Not sure it can be used as a page title